src/js/stores/DCOSStore.js

```diff
@@ -440,6 +440,27 @@ class DCOSStore extends EventEmitter {
     });
   }
 
+  get lookupTable() {
+    return this.buildFlatServiceTree(this.serviceTree);
```

Comment:
getter doesn't cache the result does it?

Reply:
@nLight Nope, we've boxed that for now.